test/go.mod

```diff
@@ -2,39 +2,22 @@ module github.com/mesosphere/kubeaddons-base-addons/test
 
 go 1.15
 
-replace k8s.io/client-go => k8s.io/client-go v0.19.0
+replace (
+	github.com/docker/docker => github.com/docker/docker v1.4.2-0.20200203170920-46ec8731fbce // locked to this version to avoid upgrades from kind that would drop our volume provisioning code. TODO: we should update/remove our volume provisioning code.
+	google.golang.org/grpc => google.golang.org/grpc v1.26.0 // held back to accomodate controller-runtime which has not updated for v1.3x.x+ at the time of writing
+	k8s.io/client-go => k8s.io/client-go v0.19.1 // locked for general sanity in k8s lib resolution
```

Comment:
I would prefer to not replace any version if it's not strictly necessary. Is anything breaking without this directive?

Reply:
Yes, if we don't pin this it will resolve to a more recent REF which may be broken, we need to pin this one.